### PR TITLE
Add CEv1.3.0

### PR DESCRIPTION
**Pull request by aws-asolidu**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ requirements:
   build:
     # dependencies needed for open source microsoft/vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute 
     # updating nodejs to >=20.0 as it's requirement for vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute
-    - nodejs >=20.0
+    - nodejs >=20.0,<21
```
Comment (aws-asolidu) [inline recipe/meta.yaml:22]: vscode also has dependency on @azure/msal-node@2.5 which expects node version 16,18, or 20

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {% set name = "sagemaker-code-editor" %}
-{% set version = "1.2.0" %}
+{% set version = "1.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
   # github repo set-up is in progress, once ready we will use the github release url here instead of the s3 url
```
Comment (aws-prayags) [inline recipe/meta.yaml:9]: nit - we can remove this now that we're moving to github

File: recipe/meta.yaml (hunk 1 above)
Comment (aws-prayags) [inline recipe/meta.yaml:22]: any specific reason we added `<21` ?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {% set name = "sagemaker-code-editor" %}
-{% set version = "1.2.0" %}
+{% set version = "1.3.0" %}
```
Comment (aws-prayags) [inline recipe/meta.yaml:2]: how are we managing branches in feedstock repo ? I think we should maintain similar branches as sagemaker-code-editor repo, that'll allow us to provide patches for older minor versions if needed

File: recipe/meta.yaml (hunk 1 above)
Comment (aws-asolidu) [reply recipe/meta.yaml:22]: vscode has dependency on @azure/msal-node@2.5 which expects node version 16,18, or 20.

**Comment by aws-asolidu** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 3 above)
Comment (aws-asolidu) [reply recipe/meta.yaml:2]: Yup that makes sense to me. I am unable to create branches on this repo, but we can create one for 1.2 and 1.3.

**Review by aws-prayags — APPROVED** [review]

**Pull request merged**